samples/OcfClientUpdate.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016, Intel Corporation.
+
+// Sample client that works with the linux server found in iotivity-constrained
+// This sample will find the resource, then periodically retrieve and update
+// the resource
+
+var client = require('ocf').client;
+
+print("Started OCF client");
+
+client.on('error', function(error) {
+  if (error.deviceId)
+    print("Error for device: " + error.deviceId);
+});
+
+function onupdate(resource) {
+    print("Resource updated:");
+    print("    deviceId: " + resource.deviceId);
+    print("    resourcePath: " + resource.resourcePath);
+    if (resource.properties != undefined) {
+        print("Resource property 'state' is " + resource.properties.state);
+    } else {
+        print("resource.properties not found");
+    }
+}
+
+client.on('update', onupdate);
+
+var light_state = true;
```

Comment:
lightState, but a state needs a state value... true is not such one

lightOn could be ok, or it should be lightState = "lights-on" or similar